doc/src/build/cli-client.md

```diff
@@ -12,32 +12,36 @@ The Sui Client CLI installs when you install Sui. See the [Install Sui](install.
+| `pay_sui`            | Pay SUI coins to recipients following specified amounts, with input coins. Length of recipients must be the same as that of amounts. The input coins also include the coin for gas payment, so no extra gas coin is required.                 |
+| `publish`            | Publish Move modules.                                                                                                                                                                                                                         |
+| `split-coin`         | Split a coin object into multiple coins.                                                                                                                                                                                                      |
+| `switch`             | Switch active address and network.                                                                                                                                                                                                            |
+| `transfer`           | Transfer object.                                                                                                                                                                                                                              |
+| `transfer-sui`       | Transfer SUI, and pay gas with the same SUI coin object. If amount is specified, transfers only the amount. If not specified, transfers the object.                                                                                           |
+| `upgrade`            | Upgrade a Move module.                                                                                                                                                                                                                        |
+| `verify-source`      | Verify local Move packages against on-chain packages, and optionally their dependencies.                                                                                                                                                      |
+| `replay-transaction` | Replay a given transaction to view transaction effects. Set environment variable MOVE_VM_STEP=1 to debug.                                                                                                                                     |
+| `replay-batch`       | Replay transactions listed in a file.                                                                                                                                                                                                         |
+| `replay-checkpoint`  | Replay all transactions in a range of checkpoints.                                                                                                                                                                                            |
```

Comment:
This seems to break the existing alphabetical order

Reply:
updated!

Reply:
I don't think it was -- I still see it out of alphabetical order in`main`.